docs/integrations/security-threat-detection/alert-logic.md

```diff
@@ -0,0 +1,64 @@
+---
+id: alert-logic
+title: Alert Logic
+sidebar_label: Alert Logic
+description: The Sumo Logic app for Alert Logic provides enhanced security operations by seamlessly combining the capabilities of Alert Logic's incident detection and response with Sumo Logic's advanced log management and analytics platform.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('/img/integrations/security-threat-detection/alert-logic-logo.png')} alt="thumbnail icon" width="100"/>
+
+The Sumo Logic app for Alert Logic provides enhanced security operations by seamlessly combining the capabilities of Alert Logic's incident detection and response with Sumo Logic's advanced log management and analytics platform. This powerful integration allows you to gain comprehensive insights into your security landscape, detect potential threats, and respond effectively to security incidents.
+
+Key features of the Alert Logic app include:
+
+- **Real-time Threat Visibility**. Consolidate and visualize Alert Logic incident data alongside other relevant logs in Sumo Logic, providing a unified view of your security posture.
+- **Incident Analysis**. Correlate Alert Logic incident data with other log sources to identify patterns, anomalies, and potential attack vectors.
+- **Custom Dashboards**. Create tailored dashboards and visualizations to monitor specific Alert Logic incident types, threat trends, and security metrics.
+- **Proactive Threat Hunting**. Leverage Sumo Logic's advanced search capabilities to proactively hunt for signs of compromise across your environment using Alert Logic incident data.
+- **Incident Response Collaboration**. Enable cross-functional teams to collaborate efficiently by sharing relevant incident data from Alert Logic within the Sumo Logic platform.
+- **Automation and Orchestration**. Build automated workflows and triggers based on Alert Logic incident data to streamline incident response processes.
+- **Enhanced Security Insights**. Gain deeper insights into your security landscape by analyzing Alert Logic incidents to elevate your security posture, accelerate incident response, and effectively protect your organization against evolving threats
+- **Rapid Incident Detection**. Leverage real-time correlation and alerts, to quickly detect and respond to potential security threats.
+- **Improved Incident Response**. Empower your security teams to respond effectively to incidents by providing them with a holistic view of threat data and streamlined collaboration tools.
+- **Customizable Monitoring**. Tailor the Alert Logic dashboard to your organization's security requirements to monitor the most relevant threat vectors.
+- **Scalable Threat Hunting**. Scale your threat-hunting efforts by leveraging Sumo Logic's powerful search capabilities to hunt for threats across large volumes of data.
+
```

Comment:
@JV0812, you might want to make all the bullet items in this list sentence case. (For example, change "Real-time Threat Visibility" to "Real-time threat visibility".) But it's not essential.